tsconfig.all.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./tsconfig.base",
+  "extends": "./tsconfig.json",
```

Comment:
I renamed your `tsconfig.base.json` to `tsconfig.json`.

The motivation was because, annoyingly, VS Code wigs out for any ts files that use decorators because the IDE is not smart enough to work out which tsconfig should be used for a source file when they are non standard names and so I think it just goes with the defaults which warns about decorators and fills files with a sea of red squiggles. 

Thus, I've been leaving base configs for libs as simply `tsconfig.json` in the root directory. Hope you don't mind.